Calling/src/app/views/CallScreen.tsx

```diff
@@ -1,62 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
 import { CommunicationUserIdentifier } from '@azure/communication-common';
 import {
+  CallAdapterLocator,
   CallAdapter,
   CallAdapterState,
   CallComposite,
-  createAzureCommunicationCallAdapter,
-  toFlatCommunicationIdentifier
+  toFlatCommunicationIdentifier,
+  useAzureCommunicationCallAdapter
 } from '@azure/communication-react';
 import { Spinner } from '@fluentui/react';
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
 import { useSwitchableFluentTheme } from '../theming/SwitchableFluentThemeProvider';
 import { createAutoRefreshingCredential } from '../utils/credential';
-import MobileDetect from 'mobile-detect';
-
-const detectMobileSession = (): boolean => !!new MobileDetect(window.navigator.userAgent).mobile();
+import { WEB_APP_TITLE } from '../utils/AppUtils';
+import { useIsMobile } from '../utils/useIsMobile';
 
 export interface CallScreenProps {
   token: string;
   userId: CommunicationUserIdentifier;
-  callLocator: GroupCallLocator | TeamsMeetingLinkLocator;
+  callLocator: CallAdapterLocator;
   displayName: string;
-  webAppTitle: string;
+  alternateCallerId?: string;
```

Comment:
`aleternateCallerId` is fishy. Is this related to the beta-only PSTN calling feature?